Formula/pycodestyle.rb

```diff
@@ -0,0 +1,15 @@
+class Pycodestyle < Formula
+  desc "Simple Python style checker in one Python file"
+  homepage "http://pycodestyle.pycqa.org"
+  url "https://github.com/PyCQA/pycodestyle/archive/2.2.0.tar.gz"
+  sha256 "aa663451c9de97d00eff396eeffe1095fd1597491341ca3c0be54983b25b1a7d"
+  head "https://github.com/PyCQA/pycodestyle.git"
+
+  def install
+    bin.install "pycodestyle.py"
+  end
+
+  test do
+    system "#{bin}/pycodestyle.py", "--help"
```

Comment:
Please write a test that does something more substantial than e.g. `--help` or `--version`. You could write a dummy Python file and assert its output on it. Something like so:
```ruby
(testpath/"dummy.py").write <<-EOS.undent
  print("hello I'm a Python file")
  # ...
EOS

assert_equal "...expected output...",
    shell_output("#{bin}/pycodestyle.py ... dummy.py")
```